jobs/reverse_log_proxy/templates/bpm.yml.erb

```diff
@@ -1,13 +1,13 @@
 <%
   ingress_addrs = []
-  if_link("doppler") { |ds|
-      ingress_addrs = ds.instances.map do |instance|
-          "#{instance.address}:#{ds.p('doppler.grpc_port')}"
-      end
-  }.else {
+  if_p("loggregator.doppler.addrs") { |daddrs|
```

Comment:
The order changed on the list, but loggregator.doppler.addrs has a default of '[]' unlike TC which has no default. Please change the default this property to not exist.